parrots.yaml

```
@@ -571,3 +571,7 @@
   tip: Blue and Green Parrots told Red Parrot something
 - gif: richpartyparrot.gif
   name: Rich Party Parrot
+- gif: brideparrot.gif
+  name: Bride Parrot
+  - gif: groomparrot.gif
```

Little extra indent here.  Fix that up an it should be good to go!
```
YAMLException: bad indentation of a sequence entry at line 576, column 3:
      - gif: groomparrot.gif
```

```suggestion
- gif: groomparrot.gif
```